display-brightness-ddcutil@themightydeity.github.com/extension.js

```diff
@@ -342,3 +341,35 @@ function openPrefs() {
         ]);
     }
 }
+
+function increase() {
+    GLib.spawn_command_line_async(`${ddcutil_path} setvcp 10 + 10 --bus 3`)
+}
+
+function decrease() {
+    GLib.spawn_command_line_async(`${ddcutil_path} setvcp 10 - 10 --bus 3`)
+}
```

Comment:
One of the problems are these functions that are probably replaceable by `setBrightness` if modified a bit or some of the functions already implemented, as you can see it needs the display parameter for `--bus`. It is set to 3 for my setup.

Reply:
In addition, quickly spawning the shortcut does not change the brightness and reaching the minimum or maximum brightness gives many errors.

Reply:
You could change this like this: increase and decrease funciton would call event emitting function with event type scroll up and scroll down event to the mainMenuButton, what you have implemented in #35 

Multiple monitors will also be handled by that and slider will also update their value accordingly